mirror.txt

```diff
@@ -676,3 +676,4 @@ registry.opensource.zalan.do/acid/pgbouncer
 registry.opensource.zalan.do/acid/postgres-operator
 registry.opensource.zalan.do/acid/spilo-14
 registry.opensource.zalan.do/acid/spilo-15
+tensorflow/tensorflow
```

Comment:
docker.io/tensorflow/tensorflow

Reply:
https://hub.docker.com/r/tensorflow/tensorflow/

This image does not belong to docker.io/

Reply:
The default is belong to docker.io/ if not domain